src/DataWrangling/ECCO/ECCO_mask.jl

```diff
@@ -40,3 +41,48 @@ end
     @inbounds mask[i, j, k] = (Tᵢ[i, j, k] == 0) 
 end
 
+struct LatitudinallyTaperedPolarMask{N, S, Z} 
+    northern_edges :: N
+    southern_edges :: S
+    z_edges :: Z
+end
+
+"""
+    LatitudinallyTaperedPolarMask(; northern_edges = (70,   75),
+                                    southern_edges = (-75, -70),
+                                    z_edges = (-20, 0))
```

Comment:
Nice! Is the proposed syntax change intentional? It wasn't mentioned in the description...

Reply:
I am actually ok also with `LinearlyTaperedPolarMask`. I just thought `Latitudinally` would express the fact that longitude doesn't play a role with the mask. However, I am happy to switch to `LinearlyTaperedPolarMask`

Reply:
yes and also adding the clarification `edges` to every argument. Somehow I think we want to make it so we don't need that clarification, perhaps by making the name of the type better. Notice that every property says "edges", suggesting that there should be a better way